backend/app/services/channels/telegram/service.py

```
@@ -0,0 +1,651 @@
+                while self._is_running:
+                    await asyncio.sleep(1)
+
+            except asyncio.CancelledError:
+                logger.info(
+                    "[Telegram] Channel %s (id=%d) polling cancelled",
+                    self.channel_name,
+                    self.channel_id,
+                )
+                break
+
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)